tests/api_tests/nodes/test_views.py

```diff
@@ -1352,7 +1467,7 @@ def test_filtering_on_invalid_field(self):
         assert_equal(res.status_code, 400)
         errors = res.json['errors']
         assert_equal(len(errors), 1)
-        assert_equal(errors[0]['detail'], 'Query string contains an invalid filter.')
+        assert_equal(errors[0]['detail'], "'invalid' is not a valid field for this endpoint.")
```

Comment:
Agree and expand on @sloria : we may not want to sanitize here: https://github.com/CenterForOpenScience/osf.io/pull/4246/files#diff-8a17f6dba621ae1374cefd4514bfc739R132

Bleach has side effects, so by sanitizing so far from the front end, we're potentially altering the query that we intend to send to the database (without improving user/application safety; Mongo doesn't care about HTML characters per se)